ml-agents-plugin-examples/setup.py

```diff
@@ -0,0 +1,11 @@
+from setuptools import setup
```

Comment:
No plans to publish this package, but we could use it to set up tests, e.g. for bad imports.